docs/source/starthere/intro.rst

```diff
@@ -80,67 +163,19 @@ Once you have :ref:`installed NeMo <installation>`, then you can run the code be
     import soundfile as sf
     sf.write("output_audio.wav", audio.to('cpu').detach().numpy()[0], 22050)
 
-You can learn more by about specific tasks you are interested in by checking out the NeMo :doc:`tutorials <./tutorials>`, or documentation (e.g. read :doc:`here <../asr/intro>` to learn more about ASR).
-
-You can also learn more about NeMo in the `NeMo Primer <https://github.com/NVIDIA/NeMo/blob/stable/tutorials/00_NeMo_Primer.ipynb>`_ tutorial, which introduces NeMo, PyTorch Lightning, and OmegaConf, and shows how to use, modify, save, and restore NeMo models. Additionally, the `NeMo Models <https://github.com/NVIDIA/NeMo/blob/stable/tutorials/01_NeMo_Models.ipynb>`__ tutorial explains the fundamentals of how NeMo models are created. These concepts are also explained in detail in the :doc:`NeMo Core <../core/core>` documentation.
-
-
-Introductory videos
--------------------
-
-See the two introductory videos below for a high level overview of NeMo.
+For detailed tutorials and documentation on specific tasks or to learn more about NeMo, check out the NeMo :doc:`tutorials <./tutorials>` or dive deeper into the documentation, such as learning about ASR in :doc:`here <../asr/intro>`.
 
-**Developing State-Of-The-Art Conversational AI Models in Three Lines of Code**
-
-.. raw:: html
-
-    <div style="position: relative; padding-bottom: 3%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
-        <iframe width="560" height="315" src="https://www.youtube.com/embed/wBgpMf_KQVw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-    </div>
-
-.. _installation:
-
-Installation
-------------
-
-The simplest way to install NeMo is via pip, see info below. 
```

Comment:
Change this sentence: "The simplest way to install NeMo is via pip, see info below. " to "The easiest method for installing NeMo is via pip. Refer to the details below.